open-metadata-implementation/access-services/data-engine/data-engine-api/src/main/java/org/odpi/openmetadata/accessservices/dataengine/model/Process.java

```diff
@@ -18,7 +18,7 @@
 @JsonInclude(JsonInclude.Include.NON_NULL)
 @JsonIgnoreProperties(ignoreUnknown = true)
 @JsonTypeInfo(use = JsonTypeInfo.Id.NONE)
-public class Process extends Asset {
+public class Process {
```

Comment:
Will this break backward compatibility?

Reply:
As was discussed on the call, DE is still in development and we should not worry about backward compatibility for this development (as also other adjustments will be done)